uwrt_mars_rover_utilities/CMakeLists.txt

```diff
@@ -0,0 +1,89 @@
+cmake_minimum_required(VERSION 3.8)
+project(uwrt_mars_rover_utilities)
+
+if (NOT CMAKE_CXX_STANDARD)
+    set(CMAKE_CXX_STANDARD 20)
+endif ()
+
+if(CMAKE_COMPILER_IS_GNUCXX OR CMAKE_CXX_COMPILER_ID MATCHES "Clang")
+  add_compile_options(-Wall -Wextra -Wpedantic)
+endif()
+
+find_package(ament_cmake REQUIRED)
+find_package(rclcpp REQUIRED)
+find_package(rclcpp_components REQUIRED)
+find_package(std_msgs REQUIRED)
+
+# include_directories(include)
+
+add_library(${PROJECT_NAME} SHARED
+    src/uwrt_can.cpp
+    #src/uwrt_params.cpp
+)
+
+target_include_directories(${PROJECT_NAME} PUBLIC
+    $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/include>
+    $<INSTALL_INTERFACE:include>
+    $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/include/uwrt_mars_rover_utilities>
+    $<INSTALL_INTERFACE:include/uwrt_mars_rover_utilities>
+)
+
+add_subdirectory(lib/uwrt_mars_rover_hw_bridge)
+
```

Comment:
This is rlly ugly. We should look for a clean fix (I can help with that, but if something breaks here in the future debugging this will not be fun)